tribe-ext-daystrip.php

```diff
@@ -28,510 +28,534 @@
-		 *
-		 * @return string
-		 * @see \Tribe\Extensions\Daystrip\Settings::set_options_prefix()
-		 *
-		 */
-		private function get_options_prefix() {
-			return (string) str_replace( '-', '_', 'tribe-ext-daystrip' );
-		}
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 */
+	public function construct() {
+		$this->add_required_plugin( 'Tribe__Events__Main', '6.0' );
+	}
 
-		/**
-		 * Get Settings instance.
-		 *
-		 * @return Settings
-		 */
-		private function get_settings() {
-			if ( empty( $this->settings ) ) {
-				$this->settings = new Settings( $this->get_options_prefix() );
-			}
+	/**
+	 * Get this plugin's options prefix.
+	 *
+	 * Settings_Helper will append a trailing underscore before each option.
+	 *
+	 * @return string
+	 * @see \Tribe\Extensions\Daystrip\Settings::set_options_prefix()
+	 *
+	 */
+	private function get_options_prefix() {
+		return (string) str_replace( '-', '_', 'tribe-ext-daystrip' );
+	}
 
-			return $this->settings;
+	/**
+	 * Get Settings instance.
+	 *
+	 * @return Settings
+	 */
+	private function get_settings() {
+		if ( empty( $this->settings ) ) {
+			$this->settings = new Settings( $this->get_options_prefix() );
 		}
 
-		/**
-		 * Extension initialization and hooks.
-		 */
-		public function init() {
-			// Load plugin textdomain
-			load_plugin_textdomain( 'tribe-ext-daystrip', false, basename( dirname( __FILE__ ) ) . '/languages/' );
+		return $this->settings;
+	}
 
-			if ( ! $this->php_version_check() ) {
-				return;
-			}
+	/**
+	 * Extension initialization and hooks.
+	 */
+	public function init() {
+		// Load plugin textdomain
+		load_plugin_textdomain( 'tribe-ext-daystrip', false, basename( dirname( __FILE__ ) ) . '/languages/' );
 
-			if ( ! $this->is_using_compatible_view_version() ) {
-				return;
-			}
+		if ( ! $this->php_version_check() ) {
+			return;
+		}
 
-			$this->class_loader();
+		// Don't run on legacy views
+		if (
+			! function_exists( 'tribe_events_views_v2_is_enabled' )
+			|| empty( tribe_events_views_v2_is_enabled() )
+		) {
+			return;
+		}
 
-			$this->get_settings();
+		$this->class_loader();
 
+		$this->get_settings();
 
-			add_filter( 'tribe_the_day_link', [ $this, 'filter_day_link' ] );
-			add_action( 'tribe_template_after_include:events/v2/day/top-bar/datepicker', [ $this, 'daystrip' ], 10, 3 );
-			add_action('wp_enqueue_scripts', [ $this, 'enquque_daystrip_styles'] );
-			add_action( 'wp_footer', [ $this, 'footer_styles' ] );
 
-			/**
-			 * @TODO Leaving here for a later version
-			 */
-			//add_filter( 'tribe_events_views_v2_view_repository_args', [ $this, 'jump_to_next_week' ], 10, 3 );
-		}
+		add_filter( 'tribe_the_day_link', [ $this, 'filter_day_link' ] );
+		add_action( 'tribe_template_after_include:events/v2/day/top-bar/datepicker', [ $this, 'daystrip' ], 10, 3 );
+		add_action('wp_enqueue_scripts', [ $this, 'enqueue_daystrip_styles'] );
```

Comment:
```suggestion
		add_action( 'wp_enqueue_scripts', [  $this, 'enqueue_daystrip_styles' ] );
```
Minor coding standard update.

Reply:
```suggestion
		add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_daystrip_styles' ] );
```